src/openzeppelin/security/pausable.cairo

```diff
@@ -0,0 +1,39 @@
+#[contract]
+mod Pausable {
+    use starknet::ContractAddress;
+    use starknet::get_caller_address;
+
+    struct Storage {
+        _paused: bool,
+    }
+
+    #[event]
+    fn Paused(account: ContractAddress) {}
+
+    #[event]
+    fn Unpaused(account: ContractAddress) {}
```

Comment:
Do we want to emit events here? In the solidity contracts, the "pausable" module is "silent".

Reply:
[is it](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/security/Pausable.sol#L89-L105)?

Reply:
My bad :/